setup.py

```diff
@@ -17,7 +17,7 @@
 
 
 _deps = [
-    "transformers>=4.34.0",
+    "transformers>=4.39.0",
```

Comment:
uhmmm you might also want to set a maximum `transformers` version :D (e.g. to `<4.41.0`) -- I'm planning to merge sample + greedy versions of the same decoding strategies together, as they only differ by 2-3 lines of code

Reply:
Thanks, resolved in https://github.com/huggingface/parler-tts/pull/43/commits/553d18f1105799fa8af05de2eecc2799966a9d39